@here/harp-examples/src/getting-started_orbiting-view.ts

```diff
@@ -48,6 +57,14 @@ export namespace CameraOrbitExample {
     gui.add(options, "globe").onChange(() => {
         map.projection = options.globe ? sphereProjection : mercatorProjection;
     });
+    gui.add(options, "headingSpeed", 0.1, 10, 0.1);
+
+    const stats = new Stats();
+    stats.domElement.style.bottom = "0px";
+    stats.domElement.style.top = "";
+    document.body.appendChild(stats.domElement);
+    map.addEventListener(MapViewEventNames.Render, stats.begin);
+    map.addEventListener(MapViewEventNames.AfterRender, stats.end);
```

Comment:
This will not cache the tile geometry creation b/c it happens outside of the rendering (i.e. using setTimeout). Nevertheless it's a good estimate for the pure rendering time after loading finished.

Reply:
Ok, good point. Do you know any way to fix this?

My primary intention for adding the stats was to make the memory usage visible, so it could be kept running and checked whether the memory is leaking.

Reply:
One workaround is to call stats.end also in the render event, just before stats.begin.
So you would count the time between two frame starts

Reply:
Good point, done.